content/config/daemon/troubleshoot.md

```diff
@@ -1,7 +1,10 @@
 ---
 title: Troubleshooting the Docker daemon
 description: Learn how to troubleshoot errors and misconfigurations in the Docker daemon
-keywords: docker, daemon, configuration, troubleshooting, error, fail to start
+keywords: |
+  docker, daemon, configuration, troubleshooting, error, fail to start,
+  networking, dns resolver, ip forwarding, dnsmasq, firewall,
+  Cannot connect to the Docker daemon. Is 'docker daemon' running on this host?
 tags: [ Troubleshooting ]
```

Comment:
Looks like we need aliases for 

- /engine/install/troubleshoot/
- /storage/troubleshooting_volume_errors/